### DISCUSSION TITLE
What exactly does "next start" do (and need to work)?

### DISCUSSION THREAD

**Discussion by FBosler** [vercel/next.js#14385] [Help]
I am wondering if there is any documentation on what exactly `next start` does (additionally to [this](https://nextjs.org/docs/advanced-features/custom-server) that is).

The reason why I am asking is that I am running my next.js app out of a docker container and am thinking it surely possible to remove some of the noise during building the container.

Cheers

**Comment by khattakdev** [comment]
It starts a Next.js Production Server.

**Reply by FBosler** [reply]
Thank you. 

Which folder does this depend on? Does it only depend on .next ?

Cheers

**Comment by khattakdev** [comment]
Yes, it lookup for .next directory which is an optimized production build and starts it.

**Reply by FBosler** [reply]
does it need anything else? or can I delete everything else and only keep .next folder?
